contracts/registry/IIdentityRegistry.sol

```diff
@@ -3,8 +3,7 @@ pragma solidity ^0.6.0;
 import "../registry/ITrustedIssuersRegistry.sol";
 import "../registry/IClaimTopicsRegistry.sol";
 
-import "@onchain-id/solidity/contracts/ClaimIssuer.sol";
-import "@onchain-id/solidity/contracts/Identity.sol";
+import "@onchain-id/solidity/contracts/IClaimIssuer.sol";
```

Comment:
`IClaimIssuer` is still importing `Identity` instead of `IIdentity` on the repo of onchainID, we will need to change all the calls to `Identity` into `IIdentity` in the TREX files once it's updated